app/views/user_mailer/email_changed.html.slim

```diff
@@ -12,4 +12,5 @@ table.hr
 
 p == t('.help',
       app: link_to(APP_NAME, Figaro.env.mailer_domain_name, class: 'gray'),
-      contact_phone_number: CONTACT_PHONE_NUMBER)
+      help_link: link_to(t('user_mailer.help_link_text'), MarketingSite.help_url),
+      contact_link: link_to(t('user_mailer.contact_link_text'), MarketingSite.contact_url))
```

Comment:
good idea! I tried putting this help text block into a partial, and the relative translations no longer worked:

> translation missing: en.user_mailer.help.help

were you suggesting just putting the vars that we pass into the translation into a shared partial? I am not sure how to do that in this situation....